docs_user/modules/proc_migrating-object-storage-data-to-rhoso-nodes.adoc

```diff
@@ -1,43 +1,38 @@
 [id="migrating-object-storage-data-to-rhoso-nodes_{context}"]
 
-= Migrating the {object_storage_first_ref} data from {OpenStackShort} to {rhos_long} nodes
+= Migrating the {object_storage} data from {OpenStackShort} to {rhos_acro} nodes
 
-To ensure availability during the {object_storage_first_ref} migration, you perform the following steps:
+The {object_storage_first_ref} migration involves the following steps:
 
-. Add new nodes to the {object_storage} rings
-. Set weights of existing nodes to 0
-. Rebalance rings, moving one replica
-. Copy rings to old nodes and restart services
-. Check replication status and repeat previous two steps until old nodes are drained
-. Remove the old nodes from the rings
+. Add new nodes to the {object_storage} rings.
+. Set weights of existing nodes to 0.
+. Rebalance rings by moving one replica.
+. Copy rings to old nodes and restart services.
+. Check replication status and repeat the previous two steps until the old nodes are drained.
+. Remove the old nodes from the rings.
 
 .Prerequisites
 
-* Previous {object_storage} adoption steps are completed.
-* No new environmental variables need to be defined, though you use the
-`CONTROLLER1_SSH` alias that was defined in a previous step.
-//kgilliga: Note to self: I'm not sure if the first 2 prereqs are necessary if no new variables need to be defined and the Object Storage adoption chapter comes before this chapter.
-* For DNS servers, all existing nodes must be able to resolve host names of the {rhocp_long} pods, for example by using the
-external IP of the DNSMasq service as name server in `/etc/resolv.conf`:
+* Adopt the {object_storage}. For more information, see xref:adopting-the-object-storage-service_adopt-control-plane[Adopting the {object_storage}].
+* For DNS servers, ensure that all existing nodes are able to resolve the hostnames of the {rhocp_long} pods, for example, by using the external IP of the DNSMasq service as the server name in `/etc/resolv.conf`:
```

Comment:
I changed "name server" to "server name" in the "For DNS servers..." prerequisite. Let me know if that is accurate.

Reply:
No, "name server" is different then server name. It should be "nameserver" most likely.